IkParameterizationType translationyaxisangle4d is not implemented

### TranslationYaxis4D is not implemented in moveit_kinematics

With the help of @gavanderhoorn his answer on http://answers.ros.org/question/263925/generating-an-ikfast-solution-for-4-dof-arm/ I have made an IkFast plugin for my robot.
When dragging the handles in Moveit the robot goal pose doesn't move, and while trying to drag I get the message:
```
[ERROR] [1500016582.397196842]: IK for this IkParameterizationType not implemented yet.
```

I've looked this up and came to the following answer on ros.answers.org http://answers.ros.org/question/191884/ik-for-this-ikparameterizationtype-not-implemented-yet/

I would be greatly helped to know which actions I should take to (similarly as in the answer above) get these kinematics implemented.

### Environment
* ROS Distro: Kinetic
* OS Version: Ubuntu 16.04

--- a/moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp
+++ b/moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp
@@ -570,15 +570,6 @@ size_t IKFastKinematicsPlugin::solve(KDL::Frame& pose_frame, const std::vector<d
       ComputeIk(trans, direction.data, vfree.size() > 0 ? &vfree[0] : nullptr, solutions);
       return solutions.GetNumSolutions();
 
-    case IKP_TranslationXAxisAngle4D:
-    case IKP_TranslationYAxisAngle4D:
-    case IKP_TranslationZAxisAngle4D:
-      // For *TranslationXAxisAngle4D*, *TranslationYAxisAngle4D*, *TranslationZAxisAngle4D* - end effector origin
-      // reaches desired 3D translation, manipulator direction makes a specific angle with x/y/z-axis (defined in the
-      // manipulator base link’s coordinate system)
-      ROS_ERROR_NAMED(name_, "IK for this IkParameterizationType not implemented yet.");
-      return 0;
-
     case IKP_TranslationLocalGlobal6D:
       // For **TranslationLocalGlobal6D**, the diagonal elements ([0],[4],[8]) are the local translation inside the end
       // effector coordinate system.
@@ -592,6 +583,10 @@ size_t IKFastKinematicsPlugin::solve(KDL::Frame& pose_frame, const std::vector<d
       ROS_ERROR_NAMED(name_, "IK for this IkParameterizationType not implemented yet.");
       return 0;
 
+    case IKP_TranslationXAxisAngle4D:
+    // For *TranslationXAxisAngle4D*, *TranslationYAxisAngle4D*, *TranslationZAxisAngle4D* - end effector origin
+    // reaches desired 3D translation, manipulator direction makes a specific angle with x/y/z-axis (defined in the
+    // manipulator base link's coordinate system)
     case IKP_TranslationXAxisAngleZNorm4D:
       double roll, pitch, yaw;
       // For **TranslationXAxisAngleZNorm4D** - end effector origin reaches desired 3D translation, manipulator
@@ -601,6 +596,7 @@ size_t IKFastKinematicsPlugin::solve(KDL::Frame& pose_frame, const std::vector<d
       ComputeIk(trans, &yaw, vfree.size() > 0 ? &vfree[0] : nullptr, solutions);
       return solutions.GetNumSolutions();
 
+    case IKP_TranslationYAxisAngle4D:
     case IKP_TranslationYAxisAngleXNorm4D:
       // For **TranslationYAxisAngleXNorm4D** - end effector origin reaches desired 3D translation, manipulator
       // direction needs to be orthogonal to x axis and be rotated at a certain angle starting from the y axis (defined
@@ -609,6 +605,7 @@ size_t IKFastKinematicsPlugin::solve(KDL::Frame& pose_frame, const std::vector<d
       ComputeIk(trans, &roll, vfree.size() > 0 ? &vfree[0] : nullptr, solutions);
       return solutions.GetNumSolutions();
 
+    case IKP_TranslationZAxisAngle4D:
     case IKP_TranslationZAxisAngleYNorm4D:
       // For **TranslationZAxisAngleYNorm4D** - end effector origin reaches desired 3D translation, manipulator
       // direction needs to be orthogonal to y axis and be rotated at a certain angle starting from the z axis (defined

